acceptance/analyzer_test.go

```diff
@@ -54,6 +54,10 @@ func TestAnalyzer(t *testing.T) {
 	defer registry.Stop(t)
 
 	os.Setenv("DOCKER_CONFIG", registry.DockerDirectory)
+	// Copy registry docker directory to analyze-image container
```

Comment:
Before, we were mounting the docker config, and (on Mac) Docker Desktop was translating the uid and gid of the directory to be the CNB user after dropping privileges.